### PR TITLE
chore: add test coverage

### PR DESCRIPTION
**Pull request by goldjacobe**
didn't actually add CI yet, so just prints out a report when you run yarn test

### CODE REVIEW

**Comment by goldjacobe** [comment]
Current dependencies on/for this PR:
* main
  * **PR #6** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/6" target="_blank">Graphite</a> 
    * **PR #7** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/7" target="_blank">Graphite</a> 
      * **PR #8** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/8" target="_blank">Graphite</a> 
        * **PR #9** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/9" target="_blank">Graphite</a> 
          * **PR #10** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/10" target="_blank">Graphite</a> 
            * **PR #12** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/12" target="_blank">Graphite</a>  👈
              * **PR #11** <a href="https://app.graphite.dev/github/pr/withgraphite/retype/11" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/withgraphite/retype/12?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -22,14 +22,18 @@
...
+    "nyc": "^15.1.0",
     "ts-mocha": "^10.0.0",
     "typescript": "^4.7.3"
   },
   "scripts": {
     "first-install": "yarn sdks vscode",
     "build": "tsc --project tsconfig.build.json",
     "lint": "eslint index.ts test.ts --quiet --ext .ts",
-    "test": "tsc --noEmit && yarn ts-mocha test.ts",
+    "test": "tsc --noEmit && yarn nyc --reporter=text-summary --reporter=lcovonly ts-mocha test.ts",
```
Comment (tomasreimers) [inline package.json:33]: You don't need the first reporter

**Review by tomasreimers — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (goldjacobe) [reply package.json:33]: i did

**Comment by goldjacobe** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/withgraphite/retype/12?utm_source=stack-comment).


Job Reference: pXLNmN6X5NmFIiJrO8xZ

**Pull request merged**